bastionzero/service/targets/database.go

```diff
@@ -17,16 +18,24 @@ const (
 
 // CreateDatabaseTargetRequest is used to create a new Database target
 type CreateDatabaseTargetRequest struct {
-	TargetName      string `json:"targetName"`
-	ProxyTargetID   string `json:"proxyTargetId"`
-	RemoteHost      string `json:"remoteHost"`
-	RemotePort      Port   `json:"remotePort"`
-	LocalPort       *Port  `json:"localPort,omitempty"`
-	LocalHost       string `json:"localHost,omitempty"`
-	IsSplitCert     bool   `json:"splitCert,omitempty"`
-	DatabaseType    string `json:"databaseType,omitempty"`
-	EnvironmentID   string `json:"environmentId,omitempty"`
-	EnvironmentName string `json:"environmentName,omitempty"`
+	TargetName    string `json:"targetName"`
+	ProxyTargetID string `json:"proxyTargetId"`
+	RemoteHost    string `json:"remoteHost"`
+	// RemotePort is required for all databases; however, for GCP-hosted databases, the
+	// value specified for Port.Value will be ignored when connecting to the database.
+	// TODO: To match REST API, change to: RemotePort *Port  `json:"remotePort,omitempty"` (to be combined with other breaking changes)
+	RemotePort Port   `json:"remotePort"`
```

Comment:
```suggestion
	RemoteHost    string `json:"remoteHost"`
	// TODO: To match REST API, change to: RemotePort *Port  `json:"remotePort,omitempty"` (to be combined with other breaking changes)
	
	// RemotePort is required for all databases; however, for GCP-hosted databases, the
	// value specified for Port.Value will be ignored when connecting to the database.
	RemotePort Port   `json:"remotePort"`
```

I think if you do something like this the TODO won't leak into the public `godoc` which might be preferred

You can run `godoc -http=:6060` to confirm that the `TODO` doesn't show up when you browse to this type

Reply:
https://github.com/bastionzero/bastionzero-sdk-go/pull/40/commits/ee15653af8bfcbdeb87b33d8230bde93cf834039